src/tools/qt-gui/src/confignode.cpp

```diff
@@ -249,6 +249,11 @@ void ConfigNode::setIsExpanded (bool value)
 	m_isExpanded = value;
 }
 
+bool ConfigNode::isRoot () const
+{
+	return m_root;
```

Comment:
Isn't it somehow possible to get the property if it is a namespace key from the key itself?

Reply:
It would be pretty easy, if every `ConfigNode` stored a `ckdb::Key *` or `kdb::Key`. But only those nodes that actually represent keys of the KDB (not the intermediate ones) do that.

It is technically possible to check the `path` (= escaped name), but this is a lot easier. We could check the `path` just once in the constructor and in `setPath()` (assuming `path` isn't modified anywhere else).

Reply:
But for our purpose we don't need every `ConfigNode` to have a key just the 'namespace' nodes? Why not add a `kdb::Key` to these?